### DISCUSSION TITLE
GECE G-60 (GE Color Effects) support

### DISCUSSION THREAD

**Discussion by bobpappas** [forkineye/ESPixelStick#690] [Q&A]
Hi,
I've purchased several GE Color Effects G-60 Pathway Lights with control box model HC-DRXRGBI.

Any recommendations to get these to work?

From http://doityourselfchristmas.com/forums/showthread.php?44356-GE-Color-effects-expanded-line/page11 I see the timing is different from other GECE lights, the poster shows the results decoded from a logic analyzer. 

Looks like I may be able to add an option with these timings and signals to the beta4 branch of the firmware.

**Comment by MartinMueller2003** [comment]
You can add support yourself or I can do it in the spring. At the moment I am setting up my show for go-live in a week.

**Reply by bobpappas** [reply]
Martin, thanks for the quick answer. I may attempt the support before spring.

**Reply by bobpappas** [reply]
@MartinMueller2003 Would you recommend using current main, or beta4 as a starting point?

**Comment by MartinMueller2003** [comment]
Current main. FYI: You can contact me on WhatsApp, Discord, Skype (in order of preference) so we can have easier communications.

**Reply by bobpappas** [reply]
Ok, sent you an email. Thanks!

**Comment by MartinMueller2003** [comment]
Is there anything more happening on this?

**Reply by bobpappas** [reply]
Hi Martin, no I've been working on other projects and will likely not get back to this at all. Feel free to close this support ticket. Thanks for your help!
